src/DocTests.jl

```diff
@@ -276,7 +276,9 @@ end
 function filter_doctests(strings::NTuple{2, AbstractString},
                          doc::Documents.Document, meta::Dict)
     meta_block_filters = get(meta, :DocTestFilters, [])
-    meta_block_filters == nothing && meta_block_filters == []
+    if meta_block_filters === nothing
```

Comment:
` meta_block_filters == nothing && (meta_block_filters = [])`

is probably what was intended

Reply:
Yes, I think so, too. However, i thought my version was a bit more readable (which seems to be fine given this typo)

Reply:
I think this is the standard way of doing it as seen by the original code. It is just as easy to write `meta_block_filters == []` in your version.

Reply:
Changed in https://github.com/JuliaDocs/Documenter.jl/pull/1696/commits/f726d74c0255c47277cc7c2d308562a44c633f0d